src/it/resources/com/google/checkstyle/test/chapter5naming/rule523methodnames/InputMethodName.java

```diff
@@ -56,6 +60,69 @@ void Testing_Foo2() {} // violation 'Method name 'Testing_Foo2' must match patte
   @Test
   void TestingFooBad() {} // violation 'Method name 'TestingFooBad' must match pattern'
 
+  @ParameterizedTest
+  void testing_foo1() {}
+
+  @ParameterizedTest
+  void testing_Foo1() {}
+
+  @ParameterizedTest
+  void testing_fOo1() {}
+
+  @ParameterizedTest
+  void testingFoo1() {}
+
+  @ParameterizedTest
+  void testingFoo_foo1() {}
+
+  @ParameterizedTest
+  void testing_01231() {}
+
+  @ParameterizedTest
+  void Testing_Foo1() {} // violation 'Method name 'Testing_Foo1' must match pattern'
+
+  @ParameterizedTest
+  void t_esting1() {} // violation 'Method name 't_esting1' must match pattern'
+
+  @ParameterizedTest
+  void _testing1() {} // violation 'Method name '_testing1' must match pattern'
+
+  @ParameterizedTest
+  void TestingFooBad1() {} // violation 'Method name 'TestingFooBad1' must match pattern'
+
+  @RepeatedTest(0)
+  void testing_foo2() {}
+
+  @RepeatedTest(0)
+  void testing_Foo2() {}
+
+  @RepeatedTest(0)
+  void testing_fOo2() {}
+
+  @RepeatedTest(0)
+  void testingFoo2() {}
+
+  @RepeatedTest(0)
+  void testingFoo_foo2() {}
+
+  @RepeatedTest(0)
+  void testing_01232() {}
+
+  @RepeatedTest(0)
+  void Testing_Foo3() {} // violation 'Method name 'Testing_Foo3' must match pattern'
+
+  @RepeatedTest(0)
+  void t_esting2() {} // violation 'Method name 't_esting2' must match pattern'
+
+  @RepeatedTest(0)
+  void _testing2() {} // violation 'Method name '_testing2' must match pattern'
+
+  @RepeatedTest(0)
+  void TestingFooBad2() {} // violation 'Method name 'TestingFooBad2' must match pattern'
+
+  @BeforeAll
+  void _testingFoooo() {} // violation 'Method name '_testingFoooo' must match pattern'
```

Comment:
added extra example to show that only annotations which contains `Test` word are considered, other annotations will not work

Reply:
This is expected, this annotation is for non test methods. This method is not a test.